test/paws_test.js

```diff
@@ -598,8 +641,7 @@ describe('Unit Tests', function() {
                     done();
                 },
                 succeed: function () {
-                    AWS.restore('DynamoDB');
-                    alserviceStub.alog.restore();
+                   
```

Comment:
could you please check here that send logs function was called 3 times.

Reply:
sure

Reply:
Checked the sendLogmsgs and sendLmcstats function called thrice